src/ast.hpp

```diff
@@ -673,6 +674,7 @@ namespace Sass {
       is_important_(ptr->is_important_),
```

Comment:
The new property is not copied when cloning here.

Reply:
Nice catch